tools/travis-build

```diff
@@ -57,12 +53,16 @@ apt-get install -y --no-install-recommends \
   ${P4C_RUNTIME_DEPS} \
   git
 
-pip3 install wheel
-pip3 install $P4C_PIP3_PACKAGES
+# we want to use Python as the default so change the symlinks
+ln -sf /usr/bin/python3 /usr/bin/python
+ln -sf /usr/bin/pip3 /usr/bin/pip
+
+pip install wheel
+pip install $P4C_PIP_PACKAGES
 
 # Build libbpf for eBPF tests.
 cd /p4c
-python3 backends/ebpf/build_libbpf
+python backends/ebpf/build_libbpf
```

Comment:
[PEP 394](https://www.python.org/dev/peps/pep-0394/) recommends using `python3` instead of `python`.